test/test_helper.rb

```diff
@@ -8,3 +8,7 @@ class ActiveSupport::TestCase
 
   # Add more helper methods to be used by all tests here...
 end
+
+ActionController::TestCase.class_eval do
+  include Devise::TestHelpers
```

Comment:
![screen shot 2017-10-27 at 11 55 23 pm](https://user-images.githubusercontent.com/20709985/32119370-a4e38f62-bb72-11e7-9373-1459cbae361e.png)
As we have to include Devise::Test::ControllerHelpers, so why not include that instead of Devise::TestHelpers ?